engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java

```diff
@@ -2069,7 +2068,8 @@ public void prepareAllNicsForMigration(final VirtualMachineProfile vm, final Dep
                     profile.setIsolationUri(Networks.IsolationType.Vlan.toUri(publicIp.getVlanTag()));
                     profile.setSecurityGroupEnabled(_networkModel.isSecurityGroupSupportedInNetwork(network));
                     profile.setName(_networkModel.getNetworkTag(vm.getHypervisorType(), network));
-                    profile.setNetworId(network.getId());
+                    profile.setNetworkRate(_networkModel.getNetworkRate(network.getId(), vm.getId()));
```

Comment:
will the network id not be needed anymore?

Reply:
@DaanHoogland 
it is still needed. see line 2072
I changed setNetworId to setNetworkId